homeassistant/components/gotify/__init__.py

```diff
@@ -0,0 +1,23 @@
+"""The gotify integration."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import discovery
+
+from .const import DOMAIN
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up gotify_temp from a config entry."""
+    hass.data[DOMAIN] = entry.as_dict()
```

Comment:
This will be overwritten if there are multiple config entries.
Use `hass.data.setdefault(DOMAIN, {})[entry.entry_id] = ...`.

Reply:
How do i get access to the entry_id again inside the get_service function? I think I tried to use this method before but realised that a blank config was being passed to my get_service function since I moved to using Config Flow.

Reply:
@milanmeu is this possible as I don't think the Notify platform has an async_setup_entry method as per documentation here: https://developers.home-assistant.io/docs/config_entries_index/#for-platforms

Reply:
> How do i get access to the entry_id again inside the get_service function? I think I tried to use this method before but realised that a blank config was being passed to my get_service function since I moved to using Config Flow.

Correct

> @milanmeu is this possible as I don't think the Notify platform has an async_setup_entry method as per documentation here: https://developers.home-assistant.io/docs/config_entries_index/#for-platforms

Correct

---

I'm sorry, I can't help you with this because my knowledge of services is not sufficient.
Can we have a service for a domain setup multiple times?